modules/ossm-config-otel-kiali.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/kiali/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-otel-kiali_{context}"]
+= Configuring {DTProductName} with Kiali
```

Comment:
Same point as above re attributes, idk if you want to use those consistently for the Kiali product name